Lab_12_Menu_DOM_CSS/script.js

```diff
@@ -0,0 +1,24 @@
+const menu = document.querySelector(".menu")
+const menuIcon = document.querySelector(".menu-icon")
+menuIcon.addEventListener("click", () => {
```

Comment:
Move the event listener to the parent element so the whole button can open the menu, not just the icon inside of the button.

```suggestion
const menuButton = document.querySelector(".menu-button")
menuButton.addEventListener("click", () => {
```